### PR TITLE
Make eva-pipeline use classes of variation-commons

### PR DESCRIPTION
**Pull request by 108krohan**
- Erase models sub-package commons/models
- Add dependency on classes from `commons-variations-core` and `commons-variations-mongo`
- Have sent a tracker.txt to track file mappings from `commons/models` to the new dependencies' model classes
- Correct a couple of tests. As of now test run status: `Tests run: 470, Failures: 37, Errors: 16, Skipped: 2` Prior to the changes, test status was : Tests run: 470, Failures: 2, Errors: 0, Skipped: 2
- Shall submit fixes them on priority.

- Build passed. Log: 
```Build log attached. 
[INFO] --- maven-jar-plugin:2.6:jar (default-jar) @ eva-pipeline ---
[INFO] Building jar: /Users/rohnkum/ga4gh/eva-pipeline/target/eva-pipeline-2.1-SNAPSHOT-7d19c72.jar
[INFO] 
[INFO] --- spring-boot-maven-plugin:1.4.1.RELEASE:repackage (default) @ eva-pipeline ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.458 s
[INFO] Finished at: 2019-03-23T23:08:39+05:30
[INFO] ------------------------------------------------------------------------
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -113,6 +113,21 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>uk.ac.ebi.eva</groupId>
+            <artifactId>variation-commons-mongodb</artifactId>
+            <version>0.7-SNAPSHOT</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/test/resources/variation-commons-mongodb-0.7-SNAPSHOT.jar</systemPath>
```
Comment (sundarvenkata-EBI) [inline pom.xml]: System path with JAR is a bad idea. As I had suggested in an email conversation on Mar 20, 2019, please use the top-level POM for **variation-commons** and bring in the dependencies.

File: pom.xml [hunk 2]
```diff
@@ -113,6 +113,21 @@
...
+
+        </dependency>
+        <dependency>
+            <groupId>uk.ac.ebi.eva</groupId>
+            <artifactId>variation-commons-core</artifactId>
+            <version>0.7-SNAPSHOT</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/test/resources/variation-commons-core-0.7-SNAPSHOT.jar</systemPath>
+        </dependency>
```
Comment (sundarvenkata-EBI) [inline pom.xml]: See https://github.com/EBIvariation/eva-pipeline/pull/149/files#r271283095

File: pom.xml (hunk 1 above)
Comment (108krohan) [reply pom.xml]: Hi Sundar, you needn't look at this PR 149 as it fails Travis checks and so I started fresh with PR 150.

Comment (sundarvenkata-EBI) [reply pom.xml]: Sorry about that! I thought this was another PR...

Comment (108krohan) [reply pom.xml]: My bad, let me just close this pull request to avoid confusion. :D

**Pull request closed without merging**